packages/reactive-element/reactive-controller.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"reactive-controller.js","sources":[],"sourcesContent":[],"names":[],"mappings":""}
```

Comment:
Remove file

Reply:
Oops! I'm going to update the gitignore and force push so these aren't in any commit.